test/common/fit_common.py

```diff
@@ -196,10 +196,11 @@ def apply_stack_config():
     stack = fitargs()['stack']
     if stack is not None:
         mkcfg().add_from_file('stack_config.json', stack)
-        if 'ora' in fitcfg():
-            fitargs()['ora'] = fitcfg()['ora']
-        else:
-            fitargs()['ora'] = 'localhost'
+        if fitargs()['ora'] =='localhost':    
+            if 'ora' in fitcfg():
+               fitargs()['ora'] = fitcfg()['ora']
+            else:
+               fitargs()['ora'] = 'localhost'
```

Comment:
This else clause seems unnecessary.  It should not get to this line unless figargs()['ora'] is already 'localhost'.